### PR TITLE
[WIP] Add PurgeCSS

### LINKED ISSUE #25: Clean CSS
It'll be good if we install something like https://www.npmjs.com/package/clean-css to clean our CSS :)

### PR DESCRIPTION
**Pull request by ricardoruwer**
Fixes #25 

Before:
image

After:
image

### CODE REVIEW

File: assets/webpack.config.js [hunk 1]
```diff
@@ -5,6 +5,28 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
...
+const PurgecssPlugin = require('purgecss-webpack-plugin');
+
+function getPlugins() {
+  let plugins = [
+    new MiniCssExtractPlugin({ filename: '../css/[name].css' }),
+    new CopyWebpackPlugin([{ from: 'static/', to: '../' }]),
+    new Webpack.ProvidePlugin({ $: 'jquery', jQuery: 'jquery', 'window.jQuery': 'jquery' })
+  ]
+
+  if (process.env.MIX_ENV == 'prod') {
```
Comment (gustavodfaguiar) [inline assets/webpack.config.js]: `MIX_ENV`?

**Comment by gustavodfaguiar** [comment]
Deu falha no check :(

File: assets/webpack.config.js (hunk 1 above)
Comment (ricardoruwer) [reply assets/webpack.config.js]: That's the environment variable to see the environment 😆 `dev`, `test` or `prod`.

https://elixir-lang.org/getting-started/mix-otp/introduction-to-mix.html#environments

**Comment by ricardoruwer** [comment]
It's not working as expected, we need to try something different.

**Pull request closed without merging**